modules/core/src/webgl/vertex-array.js

```diff
@@ -388,7 +391,7 @@ export default class VertexArray {
       // index type is saved for drawElement calls
       drawParams.elementCount = this.elements.getElementCount(this.elements.accessor);
       drawParams.isIndexed = true;
-      drawParams.indexType = this.elements.accessor.type;
+      drawParams.indexType = this.elementsAccessor.type || this.elements.accessor.type;
```

Comment:
I wonder if we should just pre-merge `this.elementsAccessor` when we set it, so we don't have to remember to apply fragile overrides everywhere?

Reply:
Agree, let me do that.